client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -58,12 +42,13 @@ class ActivityLogDay extends Component {
 	getRewindButton( type = '' ) {
 		return (
 			<Button
-				primary={ 'primary' === type }
-				disabled={ ! this.props.isRewindEnabled }
-				onClick={ this.handleRestoreClick }
 				compact
+				disabled={ ! this.props.isRewindEnabled }
+				onClick={ this.handleClickRestore }
+				primary={ 'primary' === type }
 			>
 				<Gridicon icon="history" size={ 18 } />
+				{ ' ' }
```

Comment:
It's one way of forcing React to paint white space between elements. It prevents the icon and the text from being _right_ next to each other.